aws/dubbd-aws/zarf-config.yaml

```
@@ -17,21 +16,9 @@ package:
       cert_file: bigbang.dev.cert
       # -- Name of the EKS cluster
       name: "big-bang-aws"
-      # -- Name of existing Terraform state bucket
-      state_bucket_name: uds-dev-state-bucket
-      # -- Key path to Terraform state file within the bucket
-      state_key: tfstate/dev/uds-dev-state-bucket.tfstate
-      # -- Name of DynamoDB table used for Terraform state locking
-      state_dynamodb_table_name: uds-dev-state-dynamodb
-      # -- AWS region
-      region: us-west-2
       # -- Provision a private/internal load balancer for the admin ingress gateway, if false a public load balancer will be provisioned
       private_admin_lb: true
       # -- Provision a private/internal load balancer for the tenant ingress gateway, if false a public load balancer will be provisioned
       private_tenant_lb: true
       # -- If set to true, force delete all resources on removal (i.e. loki S3 bucket, PVCs, etc)
```

```suggestion
      # -- If set to true, delete PVs on removal
```
Not sure if that's the right way to comment on this, but removing loki from this would be good.

---

@mjnagel - good call, most recent commit should address this.